.drone.yml

```diff
@@ -0,0 +1,80 @@
+---
+kind: pipeline
+type: kubernetes
+name: pr
+
+trigger:
+  event:
+  - pull_request
+
+steps:
+  - name: fetch tags
+    image: docker:git
+    commands:
+      - git fetch --tags
+  - name: build
+    image: golang:1.16
+    commands:
+      - make build
+  - name: test
+    image: golang:1.16
+    commands:
+      - make test
+  - name: lint
+    image: golangci/golangci-lint:v1.35.2
+    commands:
+      - golangci-lint run
```

Comment:
I would generally advise running lint before tests (and possibly even before build) as it gives more useful feedback earlier in the cycle.

Reply:
Moved lint before build and test in 3a06b25.